.github/workflows/images-legacy.yaml

```diff
@@ -44,7 +44,13 @@ jobs:
           - name: docker-plugin
             dockerfile: ./images/cilium-docker-plugin/Dockerfile
 
+          - name: cilium-test
+            dockerfile: ./images/cilium-test/Dockerfile
+
     steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
```

Comment:
Sorry we can't have this here. The problem is that we are trying to compile arm64 images for `cilium-test`. Having this here creates a diff in the setup of images that are released, by the `image-legacy-release.yaml` workflow, and this GH workflow. We have 2 options:
1) create a new GH workflows that builds cilium-test with qemu.
2) remove the arm64 support for cilium-test

Reply:
@aanm is the plan to create new image workflows for future arm64 images? If these workflows are supposed to be used for arm64 images too, maybe the QEMU setup will be needed for all workflows then?